matrix-sdk/src/main/java/org/matrix/androidsdk/rest/client/GroupsRestClient.java

```diff
@@ -56,7 +57,7 @@
      * {@inheritDoc}
      */
     public GroupsRestClient(HomeServerConnectionConfig hsConfig) {
-        super(hsConfig, GroupsApi.class, RestClient.URI_API_PREFIX_PATH_R0, false);
+        super(hsConfig, GroupsApi.class, RestClient.URI_API_PREFIX_PATH_R0, JsonUtils.getGson(false));
```

Comment:
can we use `GsonProvider.provideGson()` instead of `JsonUtils.getGson(false)`?

Reply:
No, I do not recommend it. The one provided by JsonUtils has some configuration specific for some Json